### ISSUE TITLE
Timecodes for "ML Zoomcamp 8.9 - Regularization and Dropout"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#152]
Youtube video: https://www.youtube.com/watch?v=74YmhVM6FTM

**Comment by dimzachar** [comment]
0:00:00 - Regularization techniques for neural network training
0:01:54 - Using dropout to randomly hide input.
0:03:45 - Dense layer dropout explained with examples.
0:05:34 - Using dropout to focus on shape.
0:07:15 - Adding dropout for regularization in neural network.
0:09:05 - Inner layer to final output with dropout.
0:11:02 - Analyzing neural network training with freezing layers.
0:13:12 - Model performance analyzed, 0.2 drop rate recommended.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
